app/javascript/react/components/Artists/ArtistTopSection.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+const ArtistTopSection = ({ name, bio, website, genre}) => {
+    return (  
+        <div className="news-image-gallery-container">
+
+            <div className="row">
+                <div className="small-12 medium-12 large-4 row">
+                    <div className="rounded-social-buttons">
+                        <div className='container'>
+                            <img className="img"src='https://images.unsplash.com/photo-1506755855567-92ff770e8d00?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=774&q=80'/>
+                        </div>
```

Comment:
unless you plan on having this adorable kitten image in your website forever, remove this until you decide on something more permanent!